openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/WheelchairFlagEncoder.java

```diff
@@ -149,36 +212,71 @@ public WheelchairFlagEncoder( int speedBits, double speedFactor ) {
         fullyWheelchairAccessibleHighways.add("residential"); // Straße im Wohngebiet
```

Comment:
Why do we have non-english comments here?

Reply:
has always been that ways since the original implementation I guess. But that thing needs major refactoring anyway, we should address that then